lib/libutee/default_bold.c

```diff
@@ -0,0 +1,1009 @@
+/*
```

Comment:
Since Jens identified the python packet needed, let's **not** commit this file, instead add the python-wand as a pre-requisite here: https://github.com/OP-TEE/optee_os/blob/master/README.md#41-prerequisites